rollup/l1/msg_storage.go

```diff
@@ -0,0 +1,262 @@
+package l1
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/core/types"
+	"github.com/scroll-tech/go-ethereum/log"
+)
+
+const (
+	defaultFetchInterval = 5 * time.Second
+)
+
+type MsgStorageState struct {
+	StartBlockHeader *types.Header
+	EndBlockHeader   *types.Header
+}
+
+type MsgStorage struct {
```

Comment:
add some metrics to MsgStorage eg: 

* whether the task is running
* which queue index is currently processed